module/Photo/view/photo/album/member-new.phtml

```diff
@@ -0,0 +1,211 @@
+<?php
```

Comment:
This seems like a lot of code duplication, can't we re-use the existing view?

Reply:
We can certainly do that.